recipe/meta.yaml

```diff
@@ -84,25 +84,23 @@
 
 {% set cuda_major = environ.get("cuda_compiler_version", "11.2").split(".")[0]|int %}
 
-{% set version = "2.7.2" %}
+{% set version = "2.8.0" %}
 
 package:
   name: libmagma
   version: {{ version }}
 
 source:
   - url: https://icl.utk.edu/projectsfiles/magma/downloads/magma-{{ version }}.tar.gz
-    sha256: 729bc1a70e518a7422fe7a3a54537a4741035a77be3349f66eac5c362576d560
+    sha256: f4e5e75350743fe57f49b615247da2cc875e5193cc90c11b43554a7c82cc4348
     patches:
       - optional-install.patch
   - path: make.inc
 
 build:
-  number: 2
+  number: 0
   skip: true  # [cuda_compiler_version == "None"]
-  skip: true  # [cuda_compiler_version == "10.2"]
-  skip: true  # [cuda_compiler_version == "11.0"]
-  skip: true  # [cuda_compiler_version == "11.1"]
+  skip: true  # [cuda_compiler_version == "11.2"]
```

Comment:
FWIW we dropped CUDA 11.2 globally ( https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/5799 )